src/components/molecules/SfGallery/README.md

```diff
@@ -0,0 +1,62 @@
+# SfGallery
+
+<!-- Write about general purpose of the component. Include screenshot (to be replaced with a live example once we migrate to vuepress) -->
+
+Swappable gallery for product images.
+
+![desktop](https://screenshotscdn.firefoxusercontent.com/images/0dec9565-4c31-45b1-9554-20b19fdf33c0.png)
+
+![desktop](https://screenshotscdn.firefoxusercontent.com/images/6234ea80-b523-4a68-9050-73d4c6ccf301.png)
+
+## Props
+
+<!--Write down props and their purpose -->
+
+- `images` - Array of image objects with the format below:
+
+```javascript
+let images = [
+  {
+    small: { url, alt },
+    big: { url, alt },
+    zoom: { url, alt }
+  }
+]
+```
+
+- `current` (1) - Selected image, default is 1 (the first image in the list).
+
+## Slots
+
+<!-- Describe slots and their purpose -->
+
+- `default` - Content on "stage", default is the selected image (big).
```

Comment:
default slot should always refer to whole content if it's not the only slot. This one should be titled "current" or similar

Reply:
By whole content you mean the component template entirely? In this case the nav (with thumbs) and the stage (with current image)...
I can do that, but I'm not sure if it's a good approach, the stage and nav are simple blocks and visual changes are easy. Replacing the content entirely seems to me more laborious, in this case may be better to just don't use the component, maybe importing only the JS file, wdyt?

Reply:
i mean whole content editable by slots. If we have more of them using default slot is usually a bad idea. The only scenario I can think of that could make sense is having the default slot for the main content and other slots for optional content that usually is not even there by default.

Reply:
So in this case we should not have a default slot at all?

Reply:
yeah